### PR TITLE
qemu.tests.block_copy: Update set_speed() to make it support more speeds.

### PR DESCRIPTION
**Pull request by CongLi**
And add more speeds test scenarios for block_stream.

Signed-off-by: Cong Li <coli@redhat.com>

id: 1370974

### CODE REVIEW

**Comment by CongLi** [comment]
17:07:16 INFO | PASS 1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.block_stream.simple_test.reset_speed.max_speed

**Comment by xutian** [comment]
looks fine.

File: qemu/tests/block_copy.py [hunk 1]
```diff
@@ -177,14 +177,15 @@ def set_speed(self):
         """
         params = self.parser_test_args()
         max_speed = params.get("max_speed")
```
Comment (suqinhuang) [inline qemu/tests/block_copy.py]: do we still need max_speed, it looks dup

Comment (CongLi) [inline qemu/tests/block_copy.py]: Yes, to keep the  compatibility of other cases, max_speed is needed.

Comment (CongLi) [inline qemu/tests/block_copy.py]: @suqinhuang
Could you help review it again?  
Thanks.

**Comment by PyLearner** [comment]
(1/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.block_stream.simple_test.reset_speed.max_speed: PASS (722.48 s)
 (2/2) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.block_stream.simple_test.reset_speed.min_speed: PASS (2034.68 s)

Acked-by : Meng Yang meyang@redhat.com

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Pull request merged**